.github/workflows/bump_submodules.yml

```diff
@@ -0,0 +1,46 @@
+name: Bump Submodule
+
+on:
+  schedule:
+    - cron: '0 0 * * 0,4'  # Run at 00:00 on Sunday and Thursday
+  pull_request:
+    branches: ["main"]
+    paths:
+      - ".github/workflows/bump_submodules.yml"
+
+jobs:
+  bump_submodule:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+      with:
+        submodules: true
+
+    - name: Bump torch-mlir submodule
+      run: |
+        cd third_party/torch-mlir
+        git fetch origin
+        git checkout origin/main
+        torch_mlir_commit=$(git rev-parse HEAD)
+        cd ../..
+        git add third_party/torch-mlir
+        git commit -m "Bump torch-mlir submodule to latest version"
```

Comment:
(ideally this would be tested on a fork of the repo before pushed here)

https://github.com/iree-org/iree/actions/runs/8899069884/job/24437557651?pr=17236#step:3:27
```
Author identity unknown
*** Please tell me who you are.
Run
  git config --global user.email "you@example.com"
  git config --global user.name "Your Name"
```

Could adapt this:
https://github.com/iree-org/iree/blob/c2571c871c0254e993a01cee92ec9152823a9db2/.github/workflows/publish_website.yml#L71-L74